datasets/amazon_us_reviews/amazon_us_reviews.py

```diff
@@ -0,0 +1,199 @@
+Each Dataset contains the following columns:
+    marketplace       - 2 letter country code of the marketplace where the review was written.
+    customer_id       - Random identifier that can be used to aggregate reviews written by a single author.
+    review_id         - The unique ID of the review.
+    product_id        - The unique Product ID the review pertains to. In the multilingual dataset the reviews
+                        for the same product in different countries can be grouped by the same product_id.
+    product_parent    - Random identifier that can be used to aggregate reviews for the same product.
+    product_title     - Title of the product.
+    product_category  - Broad product category that can be used to group reviews
+                        (also used to group the dataset into coherent parts).
+    star_rating       - The 1-5 star rating of the review.
+    helpful_votes     - Number of helpful votes.
+    total_votes       - Number of total votes the review received.
+    vine              - Review was written as part of the Vine program.
+    verified_purchase - The review is on a verified purchase.
+    review_headline   - The title of the review.
+    review_body       - The review text.
+    review_date       - The date the review was written.
+"""
+
+_DATA_OPTIONS_V1_00 = [
+    "Wireless",
+    "Watches",
+    "Video_Games",
+    "Video_DVD",
+    "Video",
+    "Toys",
+    "Tools",
+    "Sports",
+    "Software",
+    "Shoes",
+    "Pet_Products",
+    "Personal_Care_Appliances",
+    "PC",
+    "Outdoors",
+    "Office_Products",
+    "Musical_Instruments",
+    "Music",
+    "Mobile_Electronics",
+    "Mobile_Apps",
+    "Major_Appliances",
+    "Luggage",
+    "Lawn_and_Garden",
+    "Kitchen",
+    "Jewelry",
+    "Home_Improvement",
+    "Home_Entertainment",
+    "Home",
+    "Health_Personal_Care",
+    "Grocery",
+    "Gift_Card",
+    "Furniture",
+    "Electronics",
+    "Digital_Video_Games",
+    "Digital_Video_Download",
+    "Digital_Software",
+    "Digital_Music_Purchase",
+    "Digital_Ebook_Purchase",
+    "Camera",
+    "Books",
+    "Beauty",
+    "Baby",
+    "Automotive",
+    "Apparel",
+]
+
+_DATA_OPTIONS_V1_01 = ["Digital_Ebook_Purchase", "Books"]
+
+_DATA_OPTIONS_V1_02 = ["Books"]
+
+_DATA_OPTIONS = []
+
+for entry in _DATA_OPTIONS_V1_00:
+    _DATA_OPTIONS.append(entry + "_v1_00")
+
+for entry in _DATA_OPTIONS_V1_01:
+    _DATA_OPTIONS.append(entry + "_v1_01")
+
+for entry in _DATA_OPTIONS_V1_02:
+    _DATA_OPTIONS.append(entry + "_v1_02")
+
+_DL_URLS = {
+    name: "https://s3.amazonaws.com/amazon-reviews-pds/tsv/amazon_reviews_us_" + name + ".tsv.gz"
+    for name in _DATA_OPTIONS
+}
+
+
+class AmazonUSReviewsConfig(datasets.BuilderConfig):
+    """BuilderConfig for AmazonUSReviews."""
+
+    def __init__(self, *, data=None, **kwargs):
```

Comment:
why do we have the `*` in the init?

Reply:
Usually this is to prevent the use of positional args (we used that for the metrics in the lib to prevent user-mistakes) but I'll let @joeddav confirm

Reply:
This is just an artifact I missed from the original TFDS code. Maybe they used it for that reason but I don't think it's needed.